index.html

```diff
@@ -8,9 +8,103 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    <header></header>
-    <section id="home"></section>
-    <section id="benefits"></section>
+    <header>
+        <div class="header-wrapper">
+            <div class="top-menu">
+                <a href="#home" class="logo">
+                    <img src="./assets/icons/LogoEcoLiving.svg" alt="logo">
+                </a>
+                <nav>
+                    <a href="#home">Home</a>
+                    <a href="#about">About</a>
+                    <a href="">Destinations</a>
+                    <a href="">Tips</a>
```

Comment:
Suggestion to add one more <a href="">Contact us</a>

Reply:
Ok, will add section contacts linked to footer.